src/OrchardCore.Themes/TheAdmin/Assets/js/darkmode.js

```diff
@@ -1,15 +1,19 @@
-$("#btn-darkmode").click(function() {
+$('#btn-darkmode').click(function() {
     if($('#admin-darkmode').attr('media') === 'all')
     {
         $('#admin-default').attr('media', 'all');
         $('#admin-darkmode').attr('media', 'not all');
-        darkMode = false;
+        $(document.body).removeClass('darkmode');
+        $(this).children(':first').removeClass('fa-sun');
```

Comment:
No need to remove the css class, you can check my previous PR, it works well without, coz it replaces the font awesome automatically